about/contributing_datasets.md

```diff
@@ -0,0 +1,288 @@
+---
+layout: default
+title: Contributing datasets
+nav_order: 1
+parent: About Data Commons
+---
+
+# Life of a Dataset
+
+This tutorial walks through the process of structuring and inserting data into the Data Commons graph.
+
+## Background
+
+Importing data into Data Commons boils down to taking source data and converting it into a format that can be consumed into the Data Commons graph. This section presents an example that does not require statistical data.
+
+Given tabular data like the following:
+
+|country_id  |  country_name	         |  continent_id|
+|-------|--------|---------|
+|USA	     |  United States of America |  northamerica|
+|IND	     |  India                    |	        asia|
+
+
+You can represent this data as a graph via subject-predicate-object "triples" that describe the node and edge relationships.
```

Comment:
I think the concept of triples might be worth expanding upon. Or, was it sufficient enough for you as you walked through the example the first time?

Do you think it will be clearer as a table as well (then we can spell out which is the subject, predicate, object). Would be nice to spell out the India example as well, for completeness.

Reply:
I agree with you that the triples concept is rather confusing. Question: given 
```
Node: dcid:Count_MedicalTest_COVID_19_Pending
typeOf: dcs:StatisticalVariable
populationType: dcs:MedicalTest
```
and
```
Node: dcid:Count_MedicalTest_COVID_19_Pending
typeOf: dcs:StatisticalVariable
populationType: dcs:MedicalTest
statType: dcs:measuredValue
measuredProperty: dcs:count
```
what exactly are the triples in each of these definitions? Would the triple be the node, typeOf, and populationType rows put together, or would a triple instead be the node, statType, and value for the statType, considered when combined?

Reply:
Re your second comment: I personally like the arrow format, though we can absolutely go with a table if you prefer!